elements/pfe-accordion/.babelrc

```diff
@@ -1,4 +1,17 @@
 {
-  "presets": [["env", { "modules": false }]],
```

Comment:
Am I right in guessing that `modules: false` is no longer necessary in the new babel version?